po/POTFILES.in

```diff
@@ -9,6 +9,7 @@ js/app/articleHTMLRenderer.js
 js/app/buffetHistoryStore.js
 js/app/modules/banner/context.js
 js/app/modules/banner/search.js
+js/app/modules/contentGroup/contentGroup.js
```

Comment:
I think all translatable strings are contained in .iu file, which is already in listed here.